### PR TITLE
OSDOCS#8946: Adds information about the Pipelines dashboard

### PR DESCRIPTION
**Pull request by opayne1**
Adds information about the Pipelines dashboard

Version(s):
Pipelines 1.14 ?? (need to confirm whatever the next pipelines release is)

Issue:
https://issues.redhat.com/browse/OSDOCS-8946

Link to docs preview:
https://72132--docspreview.netlify.app/openshift-pipelines/latest/create/working-with-pipelines-web-console

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/op-admin-console-pipelines.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * create/working-with-pipelines-web-console.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="op-odc-pipelines-abstract_{context}"]
+= Working with {pipelines-title} in the Administrator perspective
+
+As a cluster administrator, you can view the *Pipelines Dashboard*. The *Dashboard* pages provides you with information such as:
```
Comment (mramendi) [inline modules/op-admin-console-pipelines.adoc:9]: Does the documentation tell the user how to reach this dashboard?

**Comment by aireilly** [comment]
This branch has been repointed to target pipelines-docs-main. pipelines-docs is no longer in use.

**Comment by mramendi** [comment]
I have finally reviewed this PR, sorry abut the delay.

I think that, while the PR cleans up the +Add wording from the intro where it doesn't reallyu belong, it instead introduces duplication, as teo proceduures see to be about the very same thing:

The new procedure https://72132--docspreview.netlify.app/openshift-pipelines/latest/create/working-with-pipelines-web-console#op-odc-pipelines_working-with-pipelines-web-console

The previously existing procedure https://72132--docspreview.netlify.app/openshift-pipelines/latest/create/working-with-pipelines-web-console#op-constructing-pipelines-using-pipeline-builder_working-with-pipelines-web-console

Besides they put all the Developer view procedures under the new +Add procedure and I am really not sure this structure works.

**Comment by opayne1** [comment]
Closing this PR for a reorg and doc improvements in #74677

**Pull request closed without merging**